2022-05-13 12:16:25,387 INFO  ==============================================================
2022-05-13 12:16:25,388 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
2022-05-13 12:16:25,389 INFO  Dir:   software_management_end_to_end/
2022-05-13 12:16:25,390 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
2022-05-13 12:16:25,391 INFO  ==============================================================
2022-05-13 12:16:25,400 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
2022-05-13 12:16:25,401 INFO  EnvironmentC8y Setup
2022-05-13 12:16:25,966 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-05-13 12:16:26,112 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
2022-05-13 12:16:26,113 INFO  Delaying execution by 6 seconds
2022-05-13 12:16:46,147 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 14 secs
2022-05-13 12:16:46,767 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-05-13 12:16:47,232 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-05-13 12:16:47,749 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-05-13 12:16:49,974 INFO  State of current operation: SUCCESSFUL
2022-05-13 12:16:50,200 INFO  Response status: 201
2022-05-13 12:16:50,201 INFO  Response to action: {
    "creationTime": "2022-05-13T12:16:50.155Z",
    "deviceId": "19009794",
    "deviceName": "m32sd10c",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19028896",
    "id": "19028896",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
2022-05-13 12:16:50,202 INFO  Started operation: {'creationTime': '2022-05-13T12:16:50.155Z', 'deviceId': '19009794', 'deviceName': 'm32sd10c', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19028896', 'id': '19028896', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
2022-05-13 12:16:52,415 INFO  State of operation 19028896 : PENDING
2022-05-13 12:16:54,626 INFO  State of operation 19028896 : EXECUTING
2022-05-13 12:16:56,842 INFO  State of operation 19028896 : EXECUTING
2022-05-13 12:16:59,055 INFO  State of operation 19028896 : EXECUTING
2022-05-13 12:17:01,291 INFO  State of operation 19028896 : FAILED
2022-05-13 12:17:01,292 ERROR Stopping as the operation has failed
2022-05-13 12:17:01,293 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
2022-05-13 12:17:01,316 WARN  SystemError: The operation has failed ... blocked 
2022-05-13 12:17:01,317 INFO  
2022-05-13 12:17:01,317 INFO  cleanup:
2022-05-13 12:17:05,325 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-05-13 12:17:05,888 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-05-13 12:17:05,892 INFO  
2022-05-13 12:17:05,893 INFO  Test duration: 40.52 secs
2022-05-13 12:17:05,894 INFO  Test final outcome:  BLOCKED
2022-05-13 12:17:05,895 INFO  Test outcome reason: SystemError: The operation has failed
2022-05-13 12:17:05,896 INFO  
